src/ape/api/networks.py

```diff
@@ -305,13 +305,13 @@ def encode_transaction(
         """
 
     @abstractmethod
-    def decode_logs(self, abi: EventABI, raw_logs: List[Dict]) -> Iterator[ContractLog]:
+    def decode_logs(self, logs: List[Dict], *events: EventABI) -> Iterator["ContractLog"]:
```

Comment:
the variadic argument feels a bit weird here

Reply:
@fubuloubu wanted it and I was hesitant at first, but I kind of like how it feels by not having to wrap single ABIs in a list first.

But I can go either way still, let's reach a consensus

Reply:
It's a breaking change anyways, live life on the edge 🤘